src/relax/op/tensor/datatype.cc

```diff
@@ -82,7 +83,8 @@ TVM_REGISTER_OP("relax.wrap_param")
     .set_attrs_type<WrapParamAttrs>()
     .set_num_inputs(1)
     .add_argument("data", "Tensor", "The input tensor")
-    .set_attr<FInferStructInfo>("FInferStructInfo", InferStructInfoWrapParam);
+    .set_attr<FInferStructInfo>("FInferStructInfo", InferStructInfoWrapParam)
+    .set_attr<FRelaxInferLayout>("FRelaxInferLayout", InferLayoutUnaryEwise);
```

Comment:
Do we have wrap_param at the phase of layout conversion?

Reply:
maybe